src/bin/typescript-coverage-report.ts

```diff
@@ -78,6 +84,7 @@ const options = {
   /* camelCase keys matching "long" flags in options above */
   outputDir: program.outputDir,
   threshold: program.threshold,
+  tsProjectFile: program.project,
```

Comment:
Mapping of the CLI switch `project` to the internal `tsProjectFile` as typed by `type-coverage-core` 
![image](https://user-images.githubusercontent.com/1135454/115450610-0ad97080-a214-11eb-8242-6a3d2a10b837.png)